browser/module.go

```diff
@@ -126,6 +128,9 @@ func (m *RootModule) initialize(vu k6modules.VU) {
 	if err != nil {
 		k6ext.Abort(vu.Context(), "failed to create file persister: %v", err)
 	}
+	if e, ok := initEnv.LookupEnv(env.K6TestRunID); ok && e != "" {
+		m.testRunID = e
+	}
```

Comment:
Can we test if we can get this environment variable when we set this?

Reply:
Yep, I will add an integration test in the next PR when the `windows.k6.testRunId` object is set.